tests/e2e/util.go

```diff
@@ -6100,15 +6108,15 @@ func CheckDevice(client clientset.Interface, sts *appsv1.StatefulSet, devicePath
 }
 
 // verifyDataOnRawBlockVolume helps check data integrity for raw block volumes
-func verifyDataOnRawBlockVolume(ns string, podName string, devicePath string, testData string) {
+func verifyDataOnRawBlockVolume(ns string, podName string, devicePath string, testData string, dataSizeInMB int64) {
 	//Write some data to file first and then to raw block device
-	writeDataOnRawBlockVolume(ns, podName, devicePath, testData)
+	writeDataOnRawBlockVolume(ns, podName, devicePath, testData, dataSizeInMB)
 	// Read the data to verify that is it same as what written
 	readDataFromRawBlockVolume(ns, podName, devicePath, testData)
 }
 
 // writeDataOnRawBlockVolume writes test data to raw block device
-func writeDataOnRawBlockVolume(ns string, podName string, devicePath string, testData string) {
+func writeDataOnRawBlockVolume(ns string, podName string, devicePath string, testData string, dataSizeInMB int64) {
```

Comment:
testdata is a file path or is it carrying the actual data itself

Reply:
testData is actual data that is written on device and verified on read